### ISSUE TITLE
v0.5.1 always fetches dependencies

### ISSUE THREAD

**Issue by faustinoaq** [imdrasil/jennifer.cr#137]
Hi @imdrasil I'm trying to create a Jennifer Recipe and I just realze there is something wrong with tag v0.5.1

Can you re-release/re-publish/re-create tag v0.5.1 or v0.5.2 because it doesn't include v0.5.1 on shard.yml, see:

https://github.com/imdrasil/jennifer.cr/blob/v0.5.1/shard.yml#L2

This causes problems with `shards install` because dependencies will be always broken: (notice `at`)

```
...
Installing jennifer (0.5.0 at 0.5.1)
Installing accord (1.0.0 at 1.1.0)
Using ifrit (0.1.2)
Installing time_zone (0.1.0 at 0.1.1)
...
```

(Other shards may have same issue as well)

See: https://github.com/crystal-community/jwt/issues/14

**Comment by imdrasil** [comment]
Will do 0.5.2 release (and take a look at time_zone shard version issue as well) today's evening. By the way, I've almost finished my work on introducing jennifer recipes (I believe will be able to create PR this evening as well)

**Comment by faustinoaq** [comment]
@imdrasil On, very nice :100: :+1: , please see: https://github.com/amberframework/recipes/pull/20 (feel free to contribute, even modify :wink: )

**Comment by imdrasil** [comment]
I've rereleased the 0.5.1

**Comment by faustinoaq** [comment]
@imdrasil Are you sure?

I mean, I still can see `0.5.0` here: https://github.com/imdrasil/jennifer.cr/blob/v0.5.1/shard.yml#L2 :sweat_smile:

**Comment by faustinoaq** [comment]
Did you deleted the previous v0.5.1 tag (with `0.5.0` inside `shad.yml`) ?

**Comment by imdrasil** [comment]
I deleted 0.5.1 release and added fix for the version in the shard file. ATM [last release (new 0.5.1)](https://github.com/imdrasil/jennifer.cr/releases/tag/v0.5.1) points to the correct commit.

**Comment by imdrasil** [comment]
https://github.com/imdrasil/jennifer.cr/blob/v0.5.1/shard.yml#L2 - is a link to the branch not tag

**Comment by faustinoaq** [comment]
@imdrasil Oh, ok, but I think you need to re-create the tag as well or maybe release v0.5.2 :sweat_smile: 

Because I'm still getting this:

```
...
Installing jennifer (0.5.0 at 0.5.1)
...
```

**Comment by faustinoaq** [comment]
@imdrasil Is working now :tada: I had to delete my `.cache/shards` folder :sweat_smile: 

But Jennifer dependencies are still breaking `shards install`. It is always fetching dependencies when `shards build` is used, even if `shards install` was executed successfully before, see:https://github.com/crystal-lang/shards/issues/183

```
➜  jennifertest git:(master) ✗ shards
Fetching https://github.com/imdrasil/jennifer.cr.git
Fetching https://github.com/imdrasil/sam.cr.git
Fetching https://github.com/phoffer/inflector.cr.git
Fetching https://github.com/neovintage/accord.git
Fetching https://github.com/imdrasil/ifrit.git
Fetching https://github.com/imdrasil/time_zone.git
Fetching https://github.com/TechMagister/i18n.cr.git
Installing jennifer (0.5.1)
Installing sam (0.2.4)
Installing inflector (0.1.8)
Installing accord (1.0.0 at 1.1.0)
Installing ifrit (0.1.2)
Installing time_zone (0.1.1)
Installing i18n (0.2.0 at fc96c6b12547c84da2e76495f9c970acda64976b)
```

So **accord** and **i18n** have an issue here

**Comment by faustinoaq** [comment]
By example, using granite (right dependencies tree)

```
➜  jennifertest git:(master) ✗ shards install
Fetching https://github.com/amberframework/granite.git
Fetching https://github.com/crystal-lang/crystal-db.git
Installing granite (0.11.0)
Installing db (0.5.0)
➜  jennifertest git:(master) ✗ shards build
Dependencies are satisfied
Building: jennifertest
➜  jennifertest git:(master) ✗ 
```

Notice: `Dependencies are satisfied`

**Comment by imdrasil** [comment]
yeah, I see. As I can see this happens because of accord shard. Will get rid of this during the weekends - going to implement own error list system. But this functionality will go to the 0.6.0 release.

**Comment by imdrasil** [comment]
This is fixed in #140
